lib/slack-ruby-bot/hooks/hello.rb

```diff
@@ -1,15 +1,22 @@
 module SlackRubyBot
   module Hooks
     class Hello
-      attr_accessor :logger
+      attr_accessor :logger, :connected_at
 
       def initialize(logger)
         self.logger = logger
       end
 
       def call(client, _data)
         return unless client && client.team
-        logger.info "Successfully connected team #{client.team.name} (#{client.team.id}) to https://#{client.team.domain}.slack.com."
+        log = ["Successfully #{@connected_at ? 'reconnected' : 'connected'} team #{client.team.name} (#{client.team.id}) to https://#{client.team.domain}.slack.com"]
+
+        connected_at = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+        log << "after #{connected_at - self.connected_at} seconds" if self.connected_at
```

Comment:
Should I `round` this to, say, 6 decimal places?

Reply:
I would round it up to 1 or 2 decimals and just stick an `s` at the end of it to keep it short.